docs/source/getting_started/index.rst

```diff
@@ -1,24 +1,113 @@
-graphical causal models and potential outcomes. It uses graph-based criteria and do-calculus for modeling assumptions
-and identifying a non-parametric causal effect.
+.. tab-set-code::
 
-For estimation, it switches to methods based primarily on potential outcomes and the best place to learn more about
-this is the user guide's :doc:`../user_guide/effect_inference/index` chapter. Alternatively, learn about its basic
-features in :doc:`../example_notebooks/dowhy_simple_example`.
+    .. code-block:: pip
 
-For root cause analysis features and structural analysis it uses graphical causal models and we recommend starting
-with :doc:`../user_guide/gcm_based_inference/index` in the user guide or check out
-:doc:`../example_notebooks/gcm_basic_example`.
+        pip install dowhy
 
-Finally, there's further resources available:
+    .. code-block:: conda
+
+        conda install -c conda-forge dowhy
+
+Further installation scenarios and instructions can be found at :doc:`install`.
+
+"Hello causal inference world"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this section, we will show the "Hello world" version of DoWhy. DoWhy is based on a simple unifying language for
+causal inference, unifying two powerful frameworks, namely graphical causal models (GCM) and potential outcomes (PO).
+It uses graph-based criteria and do-calculus for modeling assumptions and identifying a non-parametric causal effect.
+
+To get you started, we introduce two features out of a large variety of features DoWhy offers.
+
+Effect inference
+----------------
+
+For effect estimation, DoWhy switches to methods based primarily on potential outcomes. To do it, DoWhy offers a
+simple 4-step recipe consisting of modeling a causal mode, identification, estimation, and refutation:
+
+.. code:: python
+
+    from dowhy import CausalModel
+    import dowhy.datasets
+
+    # Generate some sample data
+    data = dowhy.datasets.linear_dataset(
+        beta=10,
+        num_common_causes=5,
+        num_instruments=2,
+        num_samples=10000)
+
+    # Step 1: Create a causal model from the data and given graph.
+    model = CausalModel(
+        data=data["df"],
+        treatment=data["treatment_name"],
+        outcome=data["outcome_name"],
+        graph=data["gml_graph"])
+
+    # Step 2: Identify causal effect and return target estimands
+    identified_estimand = model.identify_effect()
+
+    # Step 3: Estimate the target estimand using a statistical method.
+    estimate = model.estimate_effect(identified_estimand,
+                                     method_name="backdoor.propensity_score_matching")
+
+    # Step 4: Refute the obtained estimate using multiple robustness checks.
+    refute_results = model.refute_estimate(identified_estimand, estimate,
+                                           method_name="random_common_cause")
+
+No idea what this means? Then the best place to learn more about this is the user guide's :doc:`.
```

Comment:
haha, this might be too aggressive? How about, 
To understand what these four steps mean (and why we need four steps), the best place to learn more is the user's guide ... chapter. Alternatively, you can dive into the code and explore basic features in the doc notebook."

Reply:
at this point, it will also be good to highlight the fact that we interface with EconML for conditional effects, since people have found that integration useful. We can add a sentence like, 
"For estimation of conditional effects, you can also use methods from EconML using the same API, refer to this [notebook](dowhy-conditional-treatment-effects).

Reply:
Don e